docs/running-on-yarn.md

```diff
@@ -54,8 +54,8 @@ In `yarn-cluster` mode, the driver runs on a different machine than the client,
 
 # Preparations
 
-Running Spark-on-YARN requires a binary distribution of Spark which is built with YARN support.
-Binary distributions can be downloaded from the Spark project website. 
+Running Spark on YARN requires a binary distribution of Spark which is built with YARN support.
```

Comment:
Yes removing the arg is a good idea. Line 24 is just a formatting change... OK but it renders the same way. I'm not against merging some of these changes per se, just requesting maybe a different approach going forward.